tools/genvillage.js

```diff
@@ -0,0 +1,41 @@
+const commandLineArgs = require('command-line-args')
+const options = commandLineArgs([
+  { name: 'version', alias: 'v', type: String, defaultValue: '1.16.1' },
+  { name: 'bounds', alias: 'b', type: Number, multiple: true, defaultValue: [-256, 256, -256, 256] },
+  { name: 'name', alias: 'n', type: String, defaultValue: 'default' }
+])
+
+const version = options.version
+const bounds = options.bounds
+const name = options.name
+
+const World = require('prismarine-world')(version)
+const Anvil = require('prismarine-provider-anvil').Anvil(version)
+const Vec3 = require('vec3')
+const fs = require('fs-extra')
+
+fs.removeSync(`./village/${version}/world`)
```

Comment:
missing `${name}` or intended ?

Reply:
Nope! That was an accident. Thanks for the catch.